<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="E0366482" ignored="0" name="Test" tests="2" failures="1" timestamp="01 Feb 2018 09:08:45 GMT" time="0.057" errors="0">
  <testcase name="Dummy_test_to_fail" time="0.017" classname="Test.TC_001">
    <failure type="java.lang.AssertionError" message="expected [12] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [12] but found [1]
	at Test.TC_001.Dummy_test_to_fail(TC_001.java:18)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Dummy_test_to_fail -->
  <testcase name="Dummy_test_to_pass" time="0.0" classname="Test.TC_001"/>
</testsuite> <!-- Test -->
